packages/scandipwa/src/component/ProductCard/ProductCard.component.js

```diff
@@ -419,7 +425,19 @@ export class ProductCard extends PureComponent {
     }
 
     renderMainDetails() {
-        const { product: { name } } = this.props;
+        const { product: { name }, layout } = this.props;
+
+        if (layout === GRID_LAYOUT) {
```

Comment:
A lot of errors in console because  <a> cannot appear as a descendant of <a>. We render it only on PLP when layout is list mode

Reply:
Very good